data/mods/ssb/moves.ts

```diff
@@ -3424,6 +3424,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			this.add('-anim', source, "Trick", target);
 		},
 		onHit(target, source, effect) {
+			this.add(`c|${getName('Perish Song')}|/html <img src="https://i.imgflip.com/3rt1d8.png" />`);
```

Comment:
This should probably include width and height attributes?

Reply:
it doesnt particularly need it

Reply:
yeah this is fine as is tbh